docs/tutorial.rst

```diff
@@ -100,122 +135,422 @@ the ``-l`` option. (NOTE: this may a minute or so to run).
 
-The Python interface
-*****************************
+The output from a ``stdpopsim`` simulation is a *tree sequence*,
+a compact and efficient format for storing both genalogies and genome sequence.
+Some examples of analyzing tree sequences are given
+:ref:`below <sec_tute_analyses>`.
+If desired, these can be converted to VCF on the command line if the 
+`tskit <https://tskit.readthedocs.io/>`__ package is installed,
+with the ``tskit vcf`` command:
 
---------------------------
-Running a prexisting model
+.. code-block:: console
+
+   $ tskit vcf foo.ts > foo.vcf
+
+For this small example (only five samples), the file sizes are similar,
+but the tree sequence is slightly larger
+(it does carry a good bit more information about the trees, after all).
+However, if we up the sample sizes to 2000 and 3000
+(the simulation is still pretty quick)
+the tree sequence is twenty-three times smaller:
+
+.. code-block:: console
+
+   $ stdpopsim HomSap -s 1046 -g HapMapII_GRCh37 -c chr22 -o foo.ts -d OutOfAfrica_2T12 2000 3000
+   $ tskit vcf foo.ts > foo.vcf
+   $ ls -lth foo.*
+   -rw-r--r-- 1 peter peter 3139M Apr  3 10:40 foo.vcf
+   -rw-r--r-- 1 peter peter  136M Apr  3 10:39 foo.ts
+
+Zipping the files (using the `tszip <https://tszip.readthedocs.io/en/latest/>`__
+package) reduces this difference quite a lot,
+but increases time required for processing:
+
+.. code-block:: console
+
+   $ tskit vcf foo.ts | gzip -c > foo.vcf.gz
+   $ tszip foo.ts
+   $ ls -lth foo.*
+   -rw-r--r-- 1 peter peter  31M Apr  3 10:51 foo.ts.tsz
+   -rw-r--r-- 1 peter peter  72M Apr  3 10:50 foo.vcf.gz
+
+
+Using the SLiM simulation engine
+================================
+
+The default "simulation engine" -
+i.e., the program that actually does the simulating -
+is `msprime <https://msprime.readthedocs.io/en/stable/>`__,
+a coalescent simulator.
+However, it is also possible to swap this out for
+`SLiM <https://messerlab.org/slim/>`__,
+a forwards-time, individual-based simulator.
+
+Specifying the engine
+---------------------
+
+Using SLiM is as easy as passing the ``--engine/-e`` flag
+(we didn't do this above, so it used the default engine, msprime).
+For instance, to use SLiM to simulate the same 5% chunk of chromosome 22
+under the ``OutOfAfrica_2T12`` model as above,
+we would just run:
+
+.. code-block:: console
+
+    $ stdpopsim -e slim HomSap -c chr22 -l 0.05 -o foo.ts -d OutOfAfrica_2T12 2 3
+
+But: this simulation can take quite a while to run,
+so before you try that command out, read on!
+
+.. _sec_slim_scaling_factor:
+
+The scaling factor
+------------------
+
+However, even with only 5% of a chromosome,
+that is a pretty big simulation, due to the large number of individuals
+(unlike msprime, SLiM must actually simulate all the individuals in the population
+even just to get a few samples).
+To make it run fast enough for a tutorial,
+we can specify a *scaling factor*,
+described in more detail below (`sec_slim_scaling_factor`),
```

Comment:
This doesn't seem to link when pulled the branch and generated the docs. Should this use double backticks?